superset/assets/src/featureFlags.js

```diff
@@ -0,0 +1,11 @@
+// A higher-order function that takes the redux state tree and returns a
+// `isFeatureEnabled` function which takes a feature and returns whether it is enabled.
+// Note that we assume the featureFlags subtree is at the root of the redux state tree.
+export function isFeatureEnabledCreator(state) {
+  return feature => !!state.featureFlags[feature];
```

Comment:
This makes it clear to me that feature flag can only be boolean, which seems reasonable given the name. It prevents having more complex data structures like `vizTypeBlacklist = ['heatmap', 'sunburst']` .  I'm not necessarily  questioning this, just pointing it out.